scripts/create_patches_files.py

```diff
@@ -126,26 +131,26 @@ def make_api_request(url: str):
     patches = json.loads(r.text)
     return patches
 
-def get_patch_info(url: str):
+def get_patch_info(url: str, patch_id: str | None = None):
     patches = make_api_request(url)
     if isinstance(patches, list):
-        return parse_patches(patches)
+        return parse_patches(patches, patch_id)
 
     # Getting a single patch back from the API means we were invoked with a
     # particular patch id, not a time range.
     patch_id = patches["id"]
     series_url = f"https://patchwork.sourceware.org/api/1.3/series/{patches['series'][0]['id']}"
     series_info = make_api_request(series_url)
     if series_info["received_total"] > 1:
-        patch_ids = [patch['id'] for patch in series_info['patches'] if patch['id'] <= patch_id]
```

Comment:
Example series where this is not true:
https://patchwork.sourceware.org/project/gcc/list/?series=34483